README.md

```diff
@@ -39,7 +39,7 @@ but no real testing has been done to confirm this. Please file issues as appropr
 ```sh
 git clone https://github.com/chorankates/rouster.git
 cd rouster
-bundle install # use :aws to pull in fog
+bundle install # use :aws or :openstack to pull in fog
```

Comment:
this won't work unless you add an additional block in the Gemfile, which i don't recommend.

it might make more sense to add something related to line 32 above